docs/source/api/herobraine.rst

```diff
@@ -123,6 +123,14 @@ Smelt
     :undoc-members:
     :show-inheritance:
 
+Chat
```

Comment:
An update to main documentation/examples would be nice, because otherwise this feature can be missed easily :).

Maybe even a new subsection under `Hello World, you first agent` tutorial.